Documentation/HTKToMRTKPortingGuide.md

```diff
@@ -125,7 +129,7 @@ Some Utilities have been reconciled as duplicates with the Solver system. Please
 
 | HTK 2017 |  MRTK v2  |
 |----------|-----------|
-| Billboard | [`RadialView`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.RadialView) [Solver](README_Solver.md) |
+| Billboard | [`Billboard`](xref:Microsoft.MixedReality.Toolkit.UI.Billboard) (Need to add new component with new namespace.) |
```

Comment:
I think, at least, `RadialView` should be left here (in addition to `Billboard`) in case people want billboarding in the context of the solver system.
Billboarding without translation is possible with `RadialView` if angle and distance clamping are ignored, though it's currently not as configurable as Billboard wrt axis locking:
![image](https://user-images.githubusercontent.com/3580640/60045286-57d07800-9679-11e9-9e63-84b67e4419cc.png)

Reply:
@keveleigh  Interesting, that is good to know for movement configuration. I saw your other comments in #4843 to configure RadialView like billboard. The problem still is two-fold though:
1) they can't configure axis-locking for billboarding which is fairly common
2) this kind of parameter setting is non-intuitive

Even if I list RadialView as an option, developers won't know how to properly change it and then again still they will be blocked by axis locking for those interested in that. 

I feel like there needs to be a separate task to either a) upgrade billboard.cs to be part of the solver system or b) create good documentation and missing functionality to extend radialview for billboarding

Reply:
I couldn't figure out how to make the radial solver billboard very easily, neither has any of the other 2 people I talked to about this. I think it's a sign that the component is not necessarily obvious to use. I would appreciate having billboard be part of solvers, or to provide an easy way to make a component billboard using radial solver. For now to unblock people it would be good to just point to billboard, since currently people have no great way to knowing how to add billboarding. At least docs pointing to something is better than nothing, no?